### PR TITLE
RELATED: ONE-4642 ONE-4639 Input props and dropdown button customization turned off

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**
Fixes prop types of Input component
Disables customization for dropdown button
---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by ivan-nejezchleb** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1047/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/382/ : SUCCESS in 12m 00s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1046/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2635/ : SUCCESS in 18m 40s

File: libs/sdk-ui-kit/src/Form/InputPure.tsx [hunk 1]
```diff
@@ -23,8 +23,8 @@ export interface InputPureProps {
     onChange: (value: string | number, e?: React.ChangeEvent<HTMLInputElement>) => void;
     onEscKeyPress: () => void;
     onEnterKeyPress: () => void;
-    onBlur: () => void;
-    onFocus: () => void;
+    onBlur: (e?: React.FocusEvent<HTMLInputElement>) => void;
```
Comment (no23reason) [inline libs/sdk-ui-kit/src/Form/InputPure.tsx:26]: Is this argument really optional? Can the handler be called without the event present? Ditto for the following

The problem with this is that if the user wants to access the event, they would need to use `e?.target` for example, even though the `e` is aways there.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1050/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2639/ : SUCCESS in 9m 36s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/394/ : SUCCESS in 25s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/651/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2641/ : SUCCESS in 11m 50s

**Pull request merged**